### PR TITLE
 Added a quickstart for APM,BROWSER,MOBILE,INFRA

### PR DESCRIPTION
**Pull request by DarrenDoyle**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ Y] Did you check you NRQL syntax? - Does it work?
- [ Y] Did you include a Data source and Documentation reference?
- [ Y] Are all documentation links publicly accessible?
- [ Y] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [Y] Did you check your descriptive content for spelling and grammar errors?
- [ Y] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ Y] Does the PR contain a screenshot for each of your dashboards?
- [ Y] Do your screenshots show data?
- [ Y] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Comment by DarrenDoyle** [comment]
@pkudikyala FYI Let me know if I have any issues

File: quickstarts/mobile-signals /config.yml
Comment (aswanson-nr) [inline quickstarts/mobile-signals /config.yml:1]: Here as well.

File: data-sources/apm-signals /config.yml
[the remark this answers is not in the document]
Comment (DarrenDoyle) [reply data-sources/apm-signals /config.yml:1]: Fix

File: data-sources/browser-signals /config.yml
[the remark this answers is not in the document]
Comment (DarrenDoyle) [reply data-sources/browser-signals /config.yml:1]: fix

File: data-sources/mobile-signals /config.yml
[the remark this answers is not in the document]
Comment (DarrenDoyle) [reply data-sources/mobile-signals /config.yml:1]: fix

File: quickstarts/browser-signals /config.yml
[the remark this answers is not in the document]
Comment (DarrenDoyle) [reply quickstarts/browser-signals /config.yml:1]: fix

File: quickstarts/mobile-signals /config.yml
Comment (DarrenDoyle) [reply quickstarts/mobile-signals /config.yml:1]: fix

**Comment by DarrenDoyle** [comment]
thanks @aswanson-nr & @pkudikyala

File: data-sources/browser-signals/config.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+id: browser-signals
+displayName: Browser Signals
+description: |
+  Comprehensive Browser dashboard providing actionable insights, including CWV, Throughput, Errors & Latency. 
+install:
+  primary:
+    link:
+      url: https://docs.newrelic.com/docs/new-relic-solutions/best-practices-guides/full-stack-observability/browser-monitoring-best-practices-guide/
+  icon: logo.svg
```
Comment (aswanson-nr) [inline data-sources/browser-signals/config.yml:9]: The formatting is a little messed up here
```suggestion
icon: logo.svg
```

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Looks like there is a little indentation problem

**Review by aswanson-nr — APPROVED** [review]
Looks good to go!

**Review by aswanson-nr — APPROVED** [review]
LGTM. Also targeted the `release` branch instead of `main`. @DarrenDoyle just FYI, due to some changes in the publishing process, we now merge into `release` instead of targeting the `main` branch.

**Pull request merged**